.bingo/Variables.mk

```diff
@@ -29,18 +35,48 @@ $(GOIMPORTS): $(BINGO_DIR)/goimports.mod
 	@echo "(re)installing $(GOBIN)/goimports-v0.14.0"
 	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=goimports.mod -o=$(GOBIN)/goimports-v0.14.0 "golang.org/x/tools/cmd/goimports"
 
+GOJSONTOYAML := $(GOBIN)/gojsontoyaml-v0.1.0
+$(GOJSONTOYAML): $(BINGO_DIR)/gojsontoyaml.mod
+	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
+	@echo "(re)installing $(GOBIN)/gojsontoyaml-v0.1.0"
+	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=gojsontoyaml.mod -o=$(GOBIN)/gojsontoyaml-v0.1.0 "github.com/brancz/gojsontoyaml"
+
 GOLANGCI_LINT := $(GOBIN)/golangci-lint-v1.54.2
 $(GOLANGCI_LINT): $(BINGO_DIR)/golangci-lint.mod
 	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
 	@echo "(re)installing $(GOBIN)/golangci-lint-v1.54.2"
 	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=golangci-lint.mod -o=$(GOBIN)/golangci-lint-v1.54.2 "github.com/golangci/golangci-lint/cmd/golangci-lint"
 
+KIND := $(GOBIN)/kind-v0.22.0
```

Comment:
Do we need to pin `kind` via bingo here? Usually, most people have it installed already via brew or just binary, and installing via go might cause issues there. Maybe we can just install it using binaries like we did before?